crates/wysiwyg/src/dom/find_range.rs

```diff
@@ -342,4 +418,62 @@ mod test {
             panic!("Should have been a SameNodeRange: {:?}", range)
         }
     }
+
+    #[test]
+    fn finding_a_range_across_several_nodes_works() {
+        let d = cm("test<b>ing a| </b>new feature").state.dom;
+        let range = d.find_range(2, 12);
+        if let Range::MultipleNodes(r) = range {
+            // 3 text nodes + bold node
+            assert_eq!(4, r.nodes.len());
```

Comment:
We could `.to_html()` the nodes here to check we found what we think we found?

Reply:
Added `to_html()` check for each node + a helper function, is that what you meant?